code/modules/mob/living/simple_animal/eldritch_demons.dm

```diff
@@ -103,7 +103,7 @@
 	mob_linked.AdjustParalyzed(0.5 SECONDS)
 	linked_mobs -= mob_linked
 
-/mob/living/simple_animal/hostile/eldritch/raw_prophet/death(gibbed)
```

Comment:
This will only prevent runtimes when the mob is deleted by an admin for example. Eldritch mobs have `del_on_death` set true.